test/msw-api/setup-server/scenarios/cookies-request.test.ts

```diff
@@ -1,37 +1,51 @@
 import * as https from 'https'
 import { rest } from 'msw'
-import { setupServer } from 'msw/node'
+import { setupServer, SetupServerApi } from 'msw/node'
+import { ServerApi, createServer } from '@open-draft/test-server'
```

Comment:
I'm using this pull request as an opportunity to remove external HTTP servers dependency from tests and replace it with the locally running server that represents a production one (which mustn't be hit in most test cases).